json/test_mir_asdf.d

```diff
@@ -0,0 +1,33 @@
+/+ dub.sdl:
+name "test_mir_asdf"
+dependency "asdf" version="~>0.7.8"
+dependency "mir_common_json" path="mir-common-json"
+dflags "-mcpu=native" "-linkonce-templates" "-enable-cross-module-inlining" platform="ldc"
++/
+import mir_common_json;
+import asdf: deserialize;
+import std.file: read;
+
+auto calc(string text)
+{
+    return text.deserialize!Avg.coordinates.avg;
+}
+
+int main()
+{
+    auto text = cast(string) "/tmp/1.json".read;
+
+    foreach (v; [
```

Comment:
Please put verification code before reading file (and it would be even better to have a dedicated function for this).